submissions/MarharytaBoichenko/html_css_popup/index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>HTML-CSS-Popup</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <nav class="nav">
+          <ul class="nav__list list">
+            <li class="nav__item">
+              <a href="#" class="link nav__link">Mail</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="link nav__link">Images</a>
+            </li>
+          </ul>
+        </nav>
+        <ul class="user list">
+          <li class="user__item">
+            <input
+              type="checkbox"
+              class="user__checkbox"
+              name="menu"
+              id="checkbox"
```

Comment:
id is unique identifier, so better to use more specific names
https://www.w3schools.com/html/html_id.asp

Reply:
id here is not for css, it`s  for connection input-label

Reply:
Yeah, I know :)

Reply:
The value of the id attribute must be **unique** within the HTML document. 
Now imagine tones of code. `Checkbox` doesn't looks pretty unique.